app/src/main/AndroidManifest.xml

```diff
@@ -186,6 +186,13 @@
         <activity
             android:name=".feature.offer.ui.checkout.CheckoutActivity"
             android:windowSoftInputMode="adjustResize" />
+        <activity
+            android:name=".feature.crossselling.ui.CrossSellingResultActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+            android:launchMode="singleTask"
+            android:screenOrientation="portrait"
+            android:theme="@style/Hedvig.Theme"
+            android:windowSoftInputMode="adjustResize" />
```

Comment:
I simply copied the standard configuration from the rest of the activities. I assume this is what we use everywhere unless there are some specific requirements for some screens.

Reply:
That's a sensible thing to do 😉

However, this is how we 'most recently' do this 🙂

```suggestion
        <activity
            android:name=".feature.crossselling.ui.CrossSellingResultActivity"
            android:windowSoftInputMode="adjustResize" />
```

Reply:
How does it work without providing the theme to the activity? Don't we get problems with that?

Reply:
If we do not provide a theme on the `<activity />`-level, the `Activity` will use the `android:theme` defined for the `<application />` 🙂